test-it/DateTimeRecordToTimestamp.via

```diff
@@ -0,0 +1,74 @@
+// Generated on 01/06/2017 11:29:19
+define (c0 dv(.Boolean true))
+define (c1 dv(.Boolean true))
+define (c2 dv(.Boolean false))
+define (c3 dv(c(
+    e(.Double %66ractional%20second)   // Double
+    e(.Int32 %73econd)   // Int32
+    e(.Int32 %6Dinute)   // Int32
+    e(.Int32 %68our)   // Int32
+    e(.Int32 %64ay%20of%20month)   // Int32
+    e(.Int32 %6Donth)   // Int32
+    e(.Int32 %79ear)   // Int32
+    e(.Int32 %64ay%20of%20week)   // Int32
+    e(.Int32 %64ay%20of%20year)   // Int32
+    e(.Int32 %44ST)   // Int32
+) (0.123 46 23 15 5 12 2016 2 340 0 )))
+define (c14 dv(c(
+    e(.Double %66ractional%20second)   // Double
+    e(.Int32 %73econd)   // Int32
+    e(.Int32 %6Dinute)   // Int32
+    e(.Int32 %68our)   // Int32
+    e(.Int32 %64ay%20of%20month)   // Int32
+    e(.Int32 %6Donth)   // Int32
+    e(.Int32 %79ear)   // Int32
+    e(.Int32 %64ay%20of%20week)   // Int32
+    e(.Int32 %64ay%20of%20year)   // Int32
+    e(.Int32 %44ST)   // Int32
+) (0.123 46 23 21 5 12 2016 2 340 0 )))
+define (g25 dv(.String '%#m/%#d/%Y %#I:%M:%S%.3u %p'))
+define (g26 dv(.String '12/5/2016 3:23:46.123 PM'))
+define (g27 dv(.String '12/5/2016 9:23:46.123 PM'))
+//---------------------------------------------------
+//VI Definition: TestVI_DTRecordToTimestamp.gvi
+define (_%54estVI_DTRecordToTimestamp%2Egvi dv(.VirtualInstrument (
+    Locals: c(   // Data Space1
+        e(dv(.Boolean false)dataItem_Expected)   // Boolean
+        e(.Timestamp local29)   // Timestamp
+        e(.Timestamp local30)   // Timestamp
+        e(.String local31)   // String
+        e(dv(.String '')dataItem_UTCOnDateAndTimeString)   // String
+        e(.String local33)   // String
+        e(dv(.String '')dataItem_UTCOffDateAndTimeString)   // String
+        e(.Boolean local35)   // Boolean
+        e(dv(.Boolean false)dataItem_UTCOff)   // Boolean
+        e(.Boolean local37)   // Boolean
+        e(dv(.Boolean false)dataItem_UTCOn)   // Boolean
+        e(.Boolean local39)   // Boolean
+        e(dv(.Boolean false)dataItem_Actual)   // Boolean
+        e(.Boolean local41)   // Boolean
+    )
+        clump(1
+        Copy(c0 dataItem_Expected )
+        DateTimeToTimestamp(c3 c2 local29 )
+        DateTimeToTimestamp(c14 c1 local30 )
+        FormatDateTimeString(local31 g25 local30 c1 )
+        Copy(local31 dataItem_UTCOnDateAndTimeString )
+        FormatDateTimeString(local33 g25 local29 c2 )
+        Copy(local33 dataItem_UTCOffDateAndTimeString )
+        IsEQ(local33 g26 local35)
+        Copy(local35 dataItem_UTCOff )
+        IsEQ(local31 g27 local37)
+        Copy(local37 dataItem_UTCOn )
+        And(local35 local37 local39)
+        Copy(local39 dataItem_Actual )
+        //Printf ("All tests pass: '%s'.\n" dataItem_Actual)
+        Printf ("    UTC On pass: '%s'.\n" dataItem_UTCOn)
+        Printf ("        value: '%s'.\n" dataItem_UTCOnDateAndTimeString)
+        //Printf ("    UTC Off pass: '%s'.\n" dataItem_UTCOff)
```

Comment:
I actually plan on fixing this sooner rather than later, so I do not think we need to fork off tests just yet.